pymess/backend/__init__.py

```diff
@@ -64,11 +64,13 @@ def send(self, recipient, content, related_objects=None, tag=None, template=None
         relation
         :param tag: string mark that will be saved with the message
         :param template: template object from which content of the message was create
+        :param send_immediately: publishes the message regardless of the `is_turned_on_batch_sending` result
         :param kwargs: extra attributes that will be stored to the message
         """
         backend = self.get_backend(recipient)
-        message = self.create_message(recipient, content, related_objects, tag, template, **kwargs)
-        if not self.is_turned_on_batch_sending():
+        message = self.create_message(recipient=recipient, content=content, related_objects=related_objects, tag=tag,
+                                      template=template, **kwargs)
```

Comment:
It is better to pass arguments as `kwargs` because overriding `create_message` is more flexible then.